test/legacy_test/test_dropout_op.py

```diff
@@ -1874,7 +1862,7 @@ def test_static_comp(self):
                 mps.append(mp)
         for i in range(len(self.places)):
             self.assertTrue(
-                'dropout' not in [op.type for op in mps[i].block(0).ops]
+                'dropout' not in [op.name() for op in mps[i].global_block().ops]
```

Comment:
```suggestion
                'pd_op.dropout' not in [op.name() for op in mps[i].global_block().ops]
```

这里确定不是 `pd_op.dropout` 么？对于 PIR 而言 OP name 是有 dialect 前缀的，如果这里只写 `dropout` 的话，这里永远是 `True`，这个检查没有意义

Reply:
谢谢，已修改。

Reply:
本地修复一下 codestyle

Reply:
ok